js/answer.js

```diff
@@ -0,0 +1,18 @@
+/* Aufgabe: Antwort ist hidden. Mit Klick auf Show Answer, die Antwort anzeigen
+erneuter Klick auf Hide Answer, die Antwort verstecken */
```

Comment:
It's a good idea to write in a comment what exercise you want to solve. But after solving it you should remove the comment as they are not necessary for the functionality of your app. :)